src/main/java/mate/controller/AddDriverToCarController.java

```diff
@@ -0,0 +1,36 @@
+package mate.controller;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.model.Car;
+import mate.model.Driver;
+import mate.service.CarService;
+import mate.service.DriverService;
+
+public class AddDriverToCarController extends HttpServlet {
+    private static final Injector injector = Injector.getInstance("mate");
+    private final CarService carService = (CarService) injector.getInstance(CarService.class);
+    private final DriverService driverService
+            = (DriverService) injector.getInstance(DriverService.class);
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        req.getRequestDispatcher("/WEB-INF/views/addDriverToCar.jsp").forward(req, resp);
+    }
+
+    @Override
+    protected void doPost(HttpServletRequest request, HttpServletResponse response)
+            throws ServletException, IOException {
+        Long carId = Long.valueOf(request.getParameter("carId"));
+        Long driverId = Long.valueOf(request.getParameter("driverId"));
+        Car car = carService.get(carId);
+        Driver driver = driverService.get(driverId);
+        car.getDrivers().add(driver);
+        carService.update(car);

```

Comment:
lets redirect user to some page in the end (for example to page with car creation) Same in other controllers

Reply:
done